### ISSUE TITLE
Dropdown, Dropdown Group, Dropdown - Ensure color is not the sole means of communicating information - (2047840088)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5638]
### Module: 
 43 Dropdown, Dropdown Group, Dropdown 

### Violation: 
 Ensure color is not the sole means of communicating information 

### WCAG Reference: 
- [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) 

### Instance ID: 
 2047840088 

### Severity: 
 10 

### Description: 
 [ADVISORY]

[Issue]
The lack of any change, including the cursor makes the user think it doesn&#039;t do anything

[User Impact]
Ensure there is sufficient contrast between foreground and background colors. This requirement applies to text over solid and image backgrounds as well as images that contain text. Proper foreground and background contrast is necessary for users with low vision and some users with color blindness to accurately read content.

[Code Reference] 
 ```
<calcite-dropdown placement="bottom-start" scale="m" type="click" calcite-hydrated="">
      <div slot="dropdown-trigger" appearance="solid" tabindex="0">Div Trigger</div>
      <calcite-dropdown-group group-title="Div trigger" aria-label="Div trigger" role="group" selection-mode="single" calcite-hydrated="">
        <calcite-dropdown-item aria-checked="false" role="menuitemradio" tabindex="0" calcite-hydrated="">Grassland</calcite-dropdown-item>
        <calcite-dropdown-item active="" aria-checked="true" role="menuitemradio" tabindex="0" calcite-hydrated="">Forest</calcite-dropdown-item>
        <calcite-dropdown-item aria-checked="false" role="menuitemradio" tabindex="0" calcite-hydrated="">Trail</calcite-dropdown-item>
        <calcite-dropdown-item aria-checked="false" role="menuitemradio" tabindex="0" calcite-hydrated="">Park</calcite-dropdown-item>
        <calcite-dropdown-item aria-checked="false" role="menuitemradio" tabindex="0" calcite-hydrated="">Lake</calcite-dropdown-item>
        <calcite-dropdown-item aria-checked="false" role="menuitemradio" tabindex="0" calcite-hydrated="">Mountain</calcite-dropdown-item>
        <calcite-dropdown-item aria-checked="false" role="menuitemradio" tabindex="0" calcite-hydrated="">River</calcite-dropdown-item>
      </calcite-dropdown-group>
    </calcite-dropdown>
 ``` 

### Note: 
 [Suggestion]
Developers must ensure that the color contrast for all text (and images of text) meets the WCAG 2.0 Level AA requirements unless specifically excluded.

Designers or developers should modify the foreground color, background color, and/or font size so that sufficient contrast is attained.

Standard text less than 18 point (or less than 14 point if bold) must have a luminosity contrast ratio of 4.5:1 or more.

Text 18 point or larger (14 point or larger if bold) must have a luminosity contrast ratio of 3:1 or more.

Use the Level Access contrast checking tool or plugin (https://accessibility.dev/) to determine if the contrast is sufficient.

[Compliant Code Example] 
 ```

The lack of any change, including the cursor makes the user think it doesn&#039;t do anything
 ```
 

### Media Type: 
 Color and Contrast 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Corresponds to cursor behavior when triggering the dropdown from a `div`, which might be out of scope for, where the cursor remains the same when interacting with the `div`.

![div-triggered-dropdown](https://user-images.githubusercontent.com/5023024/198113892-795f2d81-8f29-4565-8aea-dcb4213fbac7.gif)
